workspaces/optic-engine/src/projections/endpoint.rs

```diff
@@ -287,6 +287,43 @@ impl EndpointProjection {
       None
     }
   }
+
+  pub fn get_response_node_indexes<'a>(
+    &'a self,
+    path_id: &'a PathComponentId,
+    method: &'a HttpMethod,
+  ) -> Option<impl Iterator<Item = NodeIndex> + 'a> {
```

Comment:
Thanks to @ejpbruel for helping me figure out how to get this function signature right. Has been something I've been scratching my head over for a while now, even avoiding iterators all together.